aio.run.runner/aio/run/runner/runner.py

```diff
@@ -207,9 +210,9 @@ async def cleanup(self) -> None:
         self._shutdown_pool()
 
     def install_reactor(self):
-        try:
+        if uvloop:
             uvloop.install()
-        except NameError:
+        else:
```

Comment:
no need to log again - i think we just want the message at the beginning on the import